packages/sdk-core/package.json

```diff
@@ -75,6 +75,7 @@
         "eslint-config-prettier": "^8.3.0",
         "eslint-plugin-eslint-comments": "^3.2.0",
         "eslint-plugin-import": "^2.25.3",
+        "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Why is this necessary? Currently SDK-Core & SDK-Redux should be linted without the ESLint plugin (if you look at the lint scripts) as per prettier's recommendation: https://prettier.io/docs/en/integrating-with-linters.html#notes

`"eslint-plugin-prettier"` is defined and used at the workspace root level though... I would leave it on that level for now.

Reply:
oops, not sure how that got in there